client/landing/stepper/declarative-flow/internals/steps-repository/__user/index.tsx

```diff
@@ -85,7 +85,10 @@ const UserStepComponent: StepType = function UserStep( {
 	};
 
 	const localeSuggestions = shouldRenderLocaleSuggestions && (
-		<LocaleSuggestions path={ window.location.pathname } locale={ locale } />
+		<LocaleSuggestions
+			path={ window.location.pathname + ( window.location.search || '' ) }
```

Comment:
`window.location.search` is an empty string by default

```suggestion
			path={ window.location.pathname + window.location.search ) }
```